src/main.js

```diff
@@ -39,8 +39,8 @@ import './StringPrototype';
 
 Vue.config.productionTip = false;
 
-new Vue({
+window.that = new Vue({
```

Comment:
```suggestion
new Vue({
```

forget to delete the debug code?